tfe/tfe_sweeper_test.go

```diff
@@ -0,0 +1,59 @@
+package tfe
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/go-tfe"
+	"log"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestMain(m *testing.M) {
+	resource.TestMain(m)
+}
+
+func getOrgSweeper(name string) *resource.Sweeper {
+	return &resource.Sweeper{
+		Name: name,
+		F: func(ununsed string) error {
+			if os.Getenv("TFE_HOSTNAME") == "" && (os.Getenv("TFE_TOKEN") == "") {
+				return fmt.Errorf("must provide environment variables TFE_HOSTNAME and TFE_TOKEN")
+			}
+			hostname := os.Getenv("TFE_HOSTNAME")
+			token := os.Getenv("TFE_TOKEN")
+			insecure := os.Getenv("TFE_INSECURE") != ""
+
+			client, err := getClient(hostname, token, insecure)
+			if err != nil {
+				return err
+			}
+
+			ctx := context.TODO()
+			orgList, err := client.Organizations.List(ctx, tfe.OrganizationListOptions{})
+			if err != nil {
+				return err
+			}
+			for _, org := range orgList.Items {
+				log.Printf("[DEBUG] Testing if org %s starts with tst-terraform or named terraform-updated", org.Name)
+				if strings.HasPrefix(org.Name, "tst-terraform") || org.Name == "terraform-updated" {
```

Comment:
It's part of a test that needs to change the org's name and check it actually worked. I will change it again anyway because I'm also working on adding random postfixes to the org name (instead of a fixed tst-terraform) for the acceptannce tests.

Reply:
yeah, totally but there's no reason the original name can't be `tst-terraform` and the updated name `tst-terraform-updated`, right?

Reply:
Sorry just read again this discussion. You are absolutely right and in the other branch I'm working on I have already changed the updated org name anyway because as it turns out that tst-terraform-updated-<random number> was hitting the name length limit!

I changed the test and now it renames `tst-terraform-<random1>` to `tst-terraform-<random2>` instead of `terraform-updated` :)